src/main/java/com/CezaryZal/exceptions/IncorrectInput.java

```diff
@@ -0,0 +1,8 @@
+package com.CezaryZal.exceptions;
+
+public class IncorrectInput extends RuntimeException {
```

Comment:
jak to exception to nazwa powinna sie konczync na ...Exception

Reply:
Ok